flake/dev/new-plugin.py

```diff
@@ -9,15 +9,18 @@
 default_nix_template = """{{ lib, ... }}:
 lib.nixvim.plugins.mkNeovimPlugin {{
   name = "{name}";
+  moduleName = "LUA_MODULE_NAME"; # TODO replace (or remove entirely if it is the same as `name`)
   packPathName = "{originalName}";
   package = "{package}";
 
-  maintainers = [ lib.maintainers.YOUR_NAME ]; # TODO
+  {maintainer_todo}
```

Comment:
Won't this leave an empty line with two spaces when `maintainer_todo == ''`?

I guess we could cheat and run treefmt and/or nixfmt on the finished file, but that seems excessive.

Should we move the `"\n  "` prefix into the `maintainer_todo` string?